app/views/handlers/fulfilment_request.py

```diff
@@ -1,8 +1,27 @@
+import json
```

Comment:
I think this class needs to be moved, as it's not a handler so could cause confusion. For now, the `data_models` folder seems the most appropriate.

Reply:
Updated.